turbopack/crates/turbopack-cli/src/contexts.rs

```diff
@@ -192,7 +193,7 @@ pub fn get_client_asset_context(
         compile_time_info,
         module_options_context,
         resolve_options_context,
-        rcstr!("client"),
+        LayerName::with_user_friendly_name(rcstr!("client"), rcstr!("Pages Router Client")),
```

Comment:
I attempted to change this to 'pages-client' as requested but this caused some obscure errors.  A few places match on the string `client` which is the issue im guessing.

Reply:
i would like to figure out some way to make these 'strings' less 'stringly'

an enum is out, but perhaps layer names themselves should be turned into 'statics' that could be widely references for these kidn of equality tests.  but then that interferes with persistence.

So the solution to that is turbo-tasks and modeling them as cells with widely known factory functions, but that introduces a lot of 'awaits'. Still i suppose that is the best solution to type safety.